### ISSUE TITLE
Custom field email template not found

### ISSUE THREAD

**Issue by stvnthomas** [verbb/formie#1400]
### Describe the bug

I've created a custom field using these two resources:

https://verbb.io/craft-plugins/formie/docs/developers/custom-field
https://verbb.io/craft-plugins/formie/user-guides/creating-your-own-custom-field-from-scratch

Everything worked great except the email template wasn't being used. I tried `getEmailHtml()` and `getEmailTemplatePath()` but neither worked. After some digging, it looks like the issue is [L196 in the Notifications model](https://github.com/verbb/formie/blob/craft-4/src/models/Notification.php#L196). L196 prepends the passed in template path with Formies template path. So, it becomes `formie/_special/email-template/custom-field-module/email-template` instead of `custom-field-module/email-template`.

I'm not sure that this is the right fix but the issue is resolved if `$component` (template path) is added to the template search paths on [L201](https://github.com/verbb/formie/blob/craft-4/src/models/Notification.php#L201).

```diff
$paths = [
    $templatePath . '.html',
    $templatePath . DIRECTORY_SEPARATOR . 'index.html',
+   $component,
];
```

### Steps to reproduce

1. Using the documentation/guide linked above, create a custom field.
2. Create a form and add the custom field. Set the form up to email submissions.
3. Add the form to the frontend and submit.
4. The resulting email will not use the email template defined in your custom field module.

### Form settings

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: Yes

### Craft CMS version

4.4.6.1

### Plugin version

2.0.26

### Multi-site?

Yes

**Comment by engram-design** [comment]
Yep, good point on the custom field providing a specific template, and I would suggest switching the following:

```diff
- $html = $notification->renderTemplate('formie-url-field/email-template', $inputOptions);
+ $html = Craft::$app->getView()->renderTemplate('formie-url-field/email-template', $inputOptions);
```

Which will work, and circumvent Formie's template resolution, but it's probably not a proper fix, as this won't resolve to a custom template in a form/email template if you wanted to override your custom field in those.

Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-4 as 2.0.26"`.

I've also updated the user guide to have this same change for getting the form template, which would've worked before, but not if you wanted to use Form Templates and a custom template for the field in there.

```diff
- $html = Craft::$app->getView()->renderTemplate('formie-url-field/form-template', $inputOptions);
+ $html = $form->renderTemplate('formie-url-field/form-template', $inputOptions);
```
